client/src/app/site/pages/meetings/pages/motions/services/export/motion-pdf-catalog.service/motion-pdf-catalog.service.ts

```diff
@@ -261,12 +261,12 @@ export class MotionPdfCatalogService {
         } else {
             state = this.motionService.getExtendedRecommendationLabel(motion);
         }
-        for (let i = 0; i < motion.submitters.length; ++i) {
-            submitterList +=
-                i !== motion.submitters.length - 1
-                    ? motion.submitters[i].getTitle() + `, `
-                    : motion.submitters[i].getTitle();
+        const submitters = motion.submitters.map(s => s.getTitle());
+        if (motion.additional_submitter) {
+            submitters.push(motion.additional_submitter);
         }
+        submitterList = submitters.join(`, `);
```

Comment:
Can't you use `motion.submitterNames` here? https://github.com/OpenSlides/openslides-client/blob/2ad927620bc7260efe01e76a2b3111dd47243ff6/client/src/app/site/pages/meetings/pages/motions/view-models/view-motion.ts#L77-L79